Exiled.API/Features/FriendlyFireStatusCode.cs

```diff
@@ -0,0 +1,35 @@
+// -----------------------------------------------------------------------
+// <copyright file="FriendlyFireStatusCode.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features
+{
+    /// <summary>
+    /// Status code for adding FriendlyFire rules.
+    /// </summary>
```

Comment:
Having some <seealso> tags that link to the API would be cool, and generate nicely on the website

Reply:
Does not seem like I can because they're in different modules
![image](https://user-images.githubusercontent.com/24619207/177080162-d3bf5384-25f8-4b79-a4a1-c3f5c1a0b044.png)

Reply:
Seealso tags go outside/after summary tags

Reply:
I don't think you understand - it does not matter if I put it there, or after the summary tags. Exiled.API is a dependency that other modules build upon, making it a transitive dependency at the Exiled.Events level. To solve this, I would have to have Exiled.API import from Exiled.Events; however, that would create a cyclic dependency which would not work. The dependency inversion principle makes it clear that a higher level class should not depend on a lower level class; therefore, the correct solution would be to have both depend on an abstraction of the co-shared data (Class, Functions, etc). This would require a significant amount of rework to allow interface abstractions. Therefore, this is not possible at the current time.

See:

![image](https://user-images.githubusercontent.com/24619207/177196442-f3ad9ba3-54ca-4827-88f0-3471f6611377.png)

Reply:
First:
<img width="299" alt="image" src="https://user-images.githubusercontent.com/79921583/177198646-2781a316-30c5-4015-8418-364f834e0124.png">

Reply:
Second: it shouldn't be placed in `API.Features`

Reply:
> Second: it shouldn't be placed in `API.Features`

Where should it be placed?

Reply:
> First: <img alt="image" width="299" src="https://user-images.githubusercontent.com/79921583/177198646-2781a316-30c5-4015-8418-364f834e0124.png"> (If that's what you mean)

Ok, Apparently I am brain dead lmao

Reply:
Using my brain, moved to Exiled.Enums

Reply:
@Thundermaker300 Sorry for my incorrect statements, I assume you want the see also to each enum xml doc, or to the enum parent doc only?

Reply:
For now I've only added the seealso to the parent doc, let me know if that's what you wanted